src/utils/error.test.ts

```diff
@@ -0,0 +1,36 @@
+import { errorCodes } from '@metamask/rpc-errors';
+
+import { isExecutionRevertedError } from './error';
+
+const executionRevertedError = {
+  code: errorCodes.rpc.invalidInput,
+  message: 'execution reverted',
+};
+
+describe('error', () => {
```

Comment:
Since you're testing `isExecutionRevertedError`, what do you think about calling this after the function name?

``` suggestion
describe('isExecutionRevertedError', () => {